src/js/models/v1/search/SearchAwardsOperation.js

```diff
@@ -169,11 +169,7 @@ class SearchAwardsOperation {
 
         // Add dateType to timePeriod object
         if (this.dateType) {
-            filters[rootKeys.timePeriod].push(
-                {
-                    [timePeriodKeys.dateType]: 'new_awards_only'
-                }
-            );
+            filters[rootKeys.timePeriod][0][timePeriodKeys.dateType] = 'new_awards_only';
```

Comment:
should you add a thing here to check if filters[rootKeys.timePeriod][0][timePeriodKeys.dateType] exists?

Reply:
We don't need to check if the .dateType field exists because that's the one we're adding with this statement. We could check to make sure filters[rootKeys.timeperiod] exists but I don't think there is a case when it will not. Even if the user does not select a time period, it gets set to all fy for the api call. And that happens before checking if dateType exists.